lib/menu.rb

```diff
@@ -0,0 +1,9 @@
+class Menu
+  attr_reader :dishes
+
```

Comment:
You could store the dishes in a class constant, if you wanted to add many more dishes to the menu, your initialize would become quite messy.